server/app/src/main/java/com/google/heartrate/wearos/app/bluetooth/server/BluetoothServer.java

```diff
@@ -0,0 +1,208 @@
+package com.google.heartrate.wearos.app.bluetooth.server;
+
+import android.bluetooth.BluetoothAdapter;
+import android.bluetooth.BluetoothDevice;
+import android.bluetooth.BluetoothGatt;
+import android.bluetooth.BluetoothGattCharacteristic;
+import android.bluetooth.BluetoothGattDescriptor;
+import android.bluetooth.BluetoothGattServer;
+import android.bluetooth.BluetoothGattService;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.util.Log;
+
+import com.google.heartrate.wearos.app.bluetooth.BluetoothUtils;
+import com.google.heartrate.wearos.app.bluetooth.server.handlers.GattServiceRequestHandler;
+import com.google.heartrate.wearos.app.gatt.GattException;
+
+import java.util.HashMap;
+import java.util.UUID;
+
+/**
+ * {@link BluetoothServer} is bluetooth gatt peripheral with a GATT Heart Rate service hosted.
```

Comment:
Try to be consistent with GATT in the documentation. I tend to have a preference for GATT.